python/adelphi/bin/adelphi

```diff
@@ -2,39 +2,60 @@
 
 import argparse
 import logging
+import os
+import os.path
 from functools import partial
 
+try:
+    from tempfile import TemporaryDirectory
+except ImportError:
+    from backports.tempfile import TemporaryDirectory
+
 import click
 
 from adelphi.cql import export_cql_schema
 from adelphi.gemini import export_gemini_schema, to_string
 from adelphi.store import with_cluster, build_keyspace_objects
+from adelphi.gh import build_github, build_repository, build_branch, commit_schemas, build_pull_request
 
 logging.basicConfig(level=logging.INFO)
 log = logging.getLogger('adelphi')
 
 @click.group()
 @click.option('--hosts', default='127.0.0.1', help='Comma-separated list of contact points. Default: 127.0.0.1')
 @click.option('--port', type=int, default=9042, help='Database RPC port. Default: 9042')
-# not implemented yet
-#@click.option('--output', metavar='/file/path', help='Output file. If not specified, it will write to stdout')
 @click.option('--username', help='Database username')
 @click.option('--password', help='Database password')
 @click.option('--keyspaces', help='Comma-separated list of keyspaces to include. If not specified, all keypaces will '
                          'be included, except system keypaces')
 @click.option('--rf', type=int, help='Replication factor to override original setting. Optional.')
 @click.option('--anonymize/--no-anonymize', default=True, help="Enable/disable schema anonymization")
+@click.option('--output-dir', help='Directory schema files should be written to. If not specified, it will write to stdout')
```

Comment:
A happy side effect: since we needed "export schemas to file" functionality for GH support anyway we get this functionality for free